Tests/LibWeb/Text/input/Streams/CountQueuingStrategy-enqueue-desired-size.html

```diff
@@ -0,0 +1,24 @@
+<script src="../include.js"></script>
```

Comment:
The test would imo be nice to have split out in a separate commit

Reply:
done and done!